src/init.js

```diff
@@ -1,27 +1,53 @@
-const { promises: fs } = require('fs');
+var fs = require('fs');
+const { promises: pfs } = require('fs');
+var https = require('https');
+var privateKey = fs.readFileSync('server.key', 'utf8');
+var certificate = fs.readFileSync('server.crt', 'utf8');
+var credentials = { key: privateKey, cert: certificate };
 const ngrok = require('ngrok');
```

Comment:
This was left in by mistake? I used yarn in this project, `yarn remove ngrok` and `yarn add open` should suffice?

Reply:
Yes, I did leave the ngrok package reference in the code by mistake.  Removed.

I am unfamiliar with yarn, but I installed and executed your suggested commands:

    yarn remove ngrok
    yarn add open

but got

    error eslint@6.8.0: The engine "node" is incompatible with this module. Expected version "^8.10.0 || ^10.13.0 || >=11.10.1". Got "11.6.0"
    error Found incompatible module.

I then ran

    yarn remove ngrok --ignore-engines
    yarn add open --ignore-engines

and everything seemed fine.

I committed the changes to package.json and yarn.lock that resulted.

Hope I did that correctly!